PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+# Pull Request Template
+
+## Description
+
+Describe the changes that where made in this pull request. When possible start with a user story - short, simple descriptions of a feature told from the perspective of the person who desires the new capability. Be sure to also include the following information:
+
+
+1. Motivation for change
+2. What was changed
+3. Link to relevant issues and documentation
+
+Example:
+*As a Blockstack developer, I would like to encrypt files using the app private key. This is needed because storing unencrypted files is unacceptable. This pull request adds the* `*encryptContent*` *function which will take a string and encrypt it using the app private key. For details refer to issue #123* 
+
+
+## Type of Change
+- [ ] New feature
+- [ ] Bug fix
+- [ ] Documentation update
+- [ ] Other
+
+
+## Does this introduce a breaking change?
+## Are documentation updates required?
```

Comment:
_Are documentation updates required_ -> How does this change impact application developers?

Then, in this section:

> * What use case does this change serve? What type of application needs it, when would developers not need to use it -- is there any precursor call or state required to use it?
> * What functional area does this change impact -- authentication, storage, collection, radiks other
> * If this is a new or change to an existing method, add an example here of the method being called

Reply:
I think this should be covered in the description section. I've added the following:
```
3. How does this impact application developers
5. Code examples for new API functions 
```